docs/checkedc/Setup-and-Build.md

```diff
@@ -107,6 +109,11 @@ Clang  needs to be placed in the tools subdirectory of LLVM.  Change to the
 ```
 git clone https://github.com/Microsoft/checkedc-clang clang
 ```
+The Checked C language tests live in a project directory for LLVM.  Change to the `llvm\projects\checkedc-wrapper` directory
```

Comment:
Why not using "the project" or just "\project" directly since it is an established directory?

Reply:
Thanks, updated.